frontend/src/components/sections/edit-mask-section.jsx

```diff
@@ -89,7 +93,7 @@ const EditMaskSection = () => {
         inExperimentId.value,
         masks,
         cameras(),
-        region.value === 'inner',
+        region.value === "inner"
```

Comment:
Please, revert the change in quotes. Look carefully at the `.prettierrc` config. Here we have 
```json
"singleQuote": true
```